From b0a68ca075928aa29ee1e0007bfb14d714f5a948 Mon Sep 17 00:00:00 2001
From: Ralf Habacker <ralf.habacker@freenet.de>
Date: Thu, 10 Aug 2017 13:45:04 +0200
Subject: Fix 'Compile error on Fedora 26'

CMake build system used some outdated variables which has been replaced
by cmake build in variables to fix the issue.

FIXED-IN:4.8.1
BUG:383351
---
 kmymoney/dialogs/settings/CMakeLists.txt | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/kmymoney/dialogs/settings/CMakeLists.txt b/kmymoney/dialogs/settings/CMakeLists.txt
index 5aacbea..b007dfb 100644
--- a/kmymoney/dialogs/settings/CMakeLists.txt
+++ b/kmymoney/dialogs/settings/CMakeLists.txt
@@ -1,12 +1,11 @@
 add_custom_command(
-  OUTPUT ${KMyMoney2_BINARY_DIR}/kmymoneysettings.h  ${KMyMoney2_BINARY_DIR}/kmymoneysettings.cpp
-  DEPENDS ${KMyMoney2_SOURCE_DIR}/kmymoney.kcfg ${KMyMoney2_SOURCE_DIR}/kmymoneysettings.kcfgc
-  COMMAND ${KDE4_KCFGC_EXECUTABLE} -d ${KMyMoney2_BINARY_DIR}/ ${KMyMoney2_SOURCE_DIR}/kmymoney.kcfg ${KMyMoney2_SOURCE_DIR}/kmymoneysettings.kcfgc
+  OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/kmymoneysettings.h  ${CMAKE_CURRENT_BINARY_DIR}/kmymoneysettings.cpp
+  DEPENDS ${CMAKE_SOURCE_DIR}/kmymoney/kmymoney.kcfg ${CMAKE_SOURCE_DIR}/kmymoney/kmymoneysettings.kcfgc
+  COMMAND ${KDE4_KCFGC_EXECUTABLE} -d ${CMAKE_CURRENT_BINARY_DIR}/ ${CMAKE_SOURCE_DIR}/kmymoney/kmymoney.kcfg ${CMAKE_SOURCE_DIR}/kmymoney/kmymoneysettings.kcfgc
   )
 
-
 set (libsettings_a_SOURCES
-  ${KMyMoney2_BINARY_DIR}/kmymoneysettings.h
+  ${CMAKE_CURRENT_BINARY_DIR}/kmymoneysettings.cpp
   ksettingscolors.cpp
   ksettingsfonts.cpp
   ksettingsforecast.cpp
-- 
cgit v0.11.2

